tutorials/en_us/Configuring-Your-Server.md

```diff
@@ -0,0 +1,268 @@
+
+### `allow-flight`
+
+When set to true, players who would normally be kicked for flying automatically will not be.
+
+### `broadcast-rcon-to-ops`
+
+Send rcon(see [`enable-rcon`](#enable-rcon)) console command outputs to all online operators.
+
+### `view-distance`
+
+The amount of world data the server sends the client.  Value is measured in chunks
+
+Possible Values: Integer (3-32) 
+
+### `max-build-height`
+
+The maximum build height that someone can build.
+
+Integer 0 - 256 (**must be multiple of 8**)
+(as of 1.16.5)
+
+### `server-ip`
+
+Set if you want the server to a particular IP.
+
+Strongly recommended to leave this blank.
+
+### `allow-nether`
+
+Allow players to teleport to the Nether dimension
+
+### `server-port`
+
+The port that the server should be hosted on
+
+### `enable-rcon`
+
+Enables remote access([RCON](https://en.wikipedia.org/wiki/Remote_administration)) to the server console.
+
+### `sync-chunk-writes`
+
+Enables synchronous chunk writes.
+
+### `op-permission-level`
+
+Sets the default permission level for ops when using `/op`. All levels inherit abilities and commands from levels before them.
+
+Possible Values:
+
+1. Ops can bypass vanilla spawn protection
+2. Ops can use all singleplayer cheats commands
+3. Ops can use most multiplayer-exclusive commands
+4. Ops can use all commands
+
+### `prevent-proxy-connections`
+
+If the ISP/AS sent from the server is different from the one from Mojang Studios' authentication server, the player is kicked
+
+### `resource-pack`
+
+URI to a resource pack
+
+Note: In some versions before 1.15.2, the ":" and "=" characters need to be escaped with a backslash (\\)
+
+### `entity-broadcast-range-percentage`
+
+Controls how close entities need to be before being sent to the client. Higher values means they'll be rendered from farther away, potentially causing more lag.
+
+Possible Values: Integer 0 - 500
+
+### `rcon.password`
+
+Password for rcon(see [`enable-rcon`](#enable-rcon))
+
+### `player-idle-timeout`
+
+The amount of minutes before a player is kicked from being idle
+
+0 means that it will not kick
+
+### `force-gamemode`
+
+Force the player into the default gamemode(see [`gamemode`](#gamemode)) upon joining
+
+### `rate-limit`
+
+Sets the maximum amount of packets a user can send before getting kicked. 
+
+0 disables this feature. 
+
+### `hardcore`
+
+When enabled, server [difficulty](#difficulty) is ignored, and set to hard. Players are set into spectator mode when they die. 
+
+### `white-list`
+
+Should whitelist be enabled.  Whitelist can be changed with the `whitelist.json` or in-game with `/whitelist`.
+
+### `broadcast-console-to-ops`
+
+Should console commands get broadcast to online Op players
+
+### `spawn-npcs`
+
+Whether or not villagers can spawn
+
+### `spawn-animals`
+
+Can animals spawn?
+
+### `snooper-enabled`
+
+Sets whether the server sends snoop data regularly to <http://snoop.minecraft.net>
```

Comment:
Maybe explain that a bit more. What is snoop data? what does snoop.minecraft.net do with it?